publish.js

```diff
@@ -473,6 +473,21 @@ var copyRecursiveSync = function(src, dest) {
     }
 };
 
+function getVersions () {
+    var topLevelOutDir = path.normalize(env.opts.destination);
```

Comment:
Yeah, Good. I agree the using git tags.
So, we have a limit of the version-switcher only to be used in git. But I think it is enough acceptable to users.

Thanks 👍